server/src/test/java/io/crate/planner/operators/JoinTest.java

```diff
@@ -369,12 +373,12 @@ public void testMixedHashJoinNestedLoop() {
                                               "left join t3 on t3.c = t2.b");
 
         LogicalPlan operator = buildLogicalPlan(mss);
-        assertThat(operator).isExactlyInstanceOf(NestedLoopJoin.class);
-        LogicalPlan leftPlan = ((NestedLoopJoin) operator).lhs;
+        assertThat(operator).isExactlyInstanceOf(HashJoin.class);
```

Comment:
This test doesn't make sense as is now, maybe we need to switch to right join, or just disable the new rule that switches nested loop to hash join, or maybe even easier just replace `=` with `>` for the left join, so that it's not an equijoin condition anymore.

Reply:
👍